src/overview/design-goals.md

```diff
@@ -19,6 +19,7 @@ We strive to start our protocol design process from user experience (UX) and use
 - The any hybrid PoW/PoS protocol (including PoW+TFL) block explorers will continue to function with the same UX through transitions in-so-far as displaying information about transactions, the mempool, and blocks.
```

Comment:
```suggestion
- Any hybrid PoW/PoS protocol (including PoW+TFL) block explorers will continue to function with the same UX through transitions in-so-far as displaying information about transactions, the mempool, and blocks.
```
Note that we can't guarantee that existing block explorers will continue to function. (We already can't guarantee that even with *no* changes to the protocol.)